Translations update from Weblate

weblate opened the pull request:
Translations update from [Weblate](https://hosted.weblate.org/projects/jasp/jasp-desktop/) for JASP/JASP desktop.



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widgets/jasp/-/jasp-desktop/horizontal-auto.svg)

JoKeyser committed:
Translated using Weblate (German)

Currently translated at 98.5% (4572 of 4640 strings)

Translation: JASP/JASP desktop
Translate-URL: https://hosted.weblate.org/projects/jasp/jasp-desktop/de/

JoKeyser committed:
Translated using Weblate (German)

Currently translated at 84.5% (1823 of 2155 strings)

Translation: JASP/JASP engine
Translate-URL: https://hosted.weblate.org/projects/jasp/jasp-engine/de/

probalino committed:
Translated using Weblate (Spanish)

Currently translated at 1.9% (93 of 4745 strings)

Translation: JASP/JASP desktop
Translate-URL: https://hosted.weblate.org/projects/jasp/jasp-desktop/es/

Commit:
Translated using Weblate (Spanish)

Currently translated at 1.9% (93 of 4745 strings)

Translation: JASP/JASP desktop
Translate-URL: https://hosted.weblate.org/projects/jasp/jasp-desktop/es/

JoKeyser committed:
Translated using Weblate (German)

Currently translated at 85.0% (1833 of 2155 strings)

Translation: JASP/JASP engine
Translate-URL: https://hosted.weblate.org/projects/jasp/jasp-engine/de/

JoKeyser committed:
Translated using Weblate (German)

Currently translated at 99.2% (4605 of 4640 strings)

Translation: JASP/JASP desktop
Translate-URL: https://hosted.weblate.org/projects/jasp/jasp-desktop/de/

JoKeyser commented:
@boutinb , just as a reminder, it would be nice to merge these contributions before the next minor release. (Generally, it would may be nice to merge regularly, in case somebody tries the nightlies or builds the stable branch before the next release.)

tomtomme committed:
Translated using Weblate (German)

Currently translated at 100.0% (4640 of 4640 strings)

Translation: JASP/JASP desktop
Translate-URL: https://hosted.weblate.org/projects/jasp/jasp-desktop/de/

tomtomme committed:
Translated using Weblate (German)

Currently translated at 100.0% (4640 of 4640 strings)

Translation: JASP/JASP desktop
Translate-URL: https://hosted.weblate.org/projects/jasp/jasp-desktop/de/

Commit:
Translated using Weblate (Spanish)

Currently translated at 3.9% (186 of 4745 strings)

Translation: JASP/JASP desktop
Translate-URL: https://hosted.weblate.org/projects/jasp/jasp-desktop/es/

probalino committed:
Translated using Weblate (Spanish)

Currently translated at 3.9% (186 of 4745 strings)

Translation: JASP/JASP desktop
Translate-URL: https://hosted.weblate.org/projects/jasp/jasp-desktop/es/

JoKeyser commented on JASP-Desktop/po/jasp_de.po:
```diff
@@ -1493,7 +1495,7 @@ msgstr "Kontraste"
 msgctxt "Ancova|"
 msgid "Confidence intervals"
-msgstr "Konfidenzintervalle"
+msgstr "Vertrauensbereich"
```
@tomtomme , in my opinion, "Konfidenzintervalle" is the more sensible and accurate translation. Its also closer to English, and is sometimes abbreviated as "KI".
It's also useful to distinguish this frequentist term from the Bayesian credibility interval.

tomtomme replied:
Hi

thanks for your feedback. I will try to explain.

For me, a translation that is near to its orginial is not neccessarily a good one. It sounds more like a non-translation. Only experts will immediately understand it. Non-Experts are pushed back. As JASP has a focus on learners and teaching its important to keep it simple. This is my perspective as a statistics lecturer.

If you like Konfidenzintervall more (or confidence intervall) then you might as well be already an expert and you can use the english version already. Someone that changes JASP from english to german is more probably not an expert but a learner. So I want to make it easy for those people.

Your second argument regarding KI - I totally get it. Konfidenzintervall is very commonly used and so is KI. But as often as that I see the abbreviation CI also used within the german language.

So my entire argument to translation is:
If you translate at all, then translate it to something a newcomer can easily understand. Otherwise, do not bother translating at all and just get used to the english words.

I would be interested in your perspective in more detail. Whats your scientific background, whats the intended target audience of JASP from your perspective etc. Maybe that way we can understand whats the best choice in the end.

Thanks, Thomas

boutinb replied:
For the Dutch translation, the goal was to have the same translation as in the Dutch Text book.
But maybe @EJWagenmakers can say more about that.

JoKeyser replied:
Hi @tomtomme,

thanks for your explanation of your point of view. I really think it is important to have this discussion.

I agree that one of JASP's goals is to be approachable for beginners, but I disagree that a common statistical term like "Konfidenzintervall" creates a significant barrier.
In my experience as researcher and university teacher of introductory statistics courses (in the social sciences), I would expect JASP to reflect the terms used in textbooks on statistics.
For example, the term "Vertrauensbereich" does not appear in the (arguably) well-known textbook Bortz & Schuster (2010), at least not in that edition.

Also, another goal of JASP is to create statistics and figures that can be used in scientific work and even publications. Also in this context, I feel that more popular and precise terms are better than more obscure and "beginner-friendly" translations.

Another argument would be that users of other statistical software, like SPSS, should not find different terms for the same concept.

- Bortz, J., & Schuster, C. (2010). Statistik für Human- und Sozialwissenschaftler. Springer Berlin Heidelberg. https://doi.org/10.1007/978-3-642-12770-0

JoKeyser replied:
@EJWagenmakers , thank you for chiming in as well. I agree that the help files would be a great place to add alternative terms.
Actually, the [German Wikipedia article on Konfidenzintervall](https://de.wikipedia.org/wiki/Konfidenzintervall) does exactly this: "Ein **Konfidenzintervall**, kurz KI, (auch **Vertrauensintervall**, **Vertrauensbereich** oder **Erwartungsbereich** genannt) ...".

More generally, I guess these kinds of discussions are likely to happen among translation teams; maybe you and @FransMeerhoff could even write some general guidelines? This kind of weighting of JASPs goals should be a top-down decision, I feel.

EJWagenmakers replied:
We'll keep track of these issues as they arise. For now, @FransMeerhoff, can you add to the instructions "When in doubt, please use the statistical terms as they occur in introductory statistics textbooks and courses in your language. We would like users to recognize the terms that are used. If you deem this useful you can mention alternative terms in the help file."

tomtomme replied:
Thanks for your perspectives. I think the help-text are a very good place for simpler terms like this! 
I will roll back the original translation in weblate and will work with you on the help-text translations. That will be fun! :D
If in doubt, maybe a guideline could be to do a german-only (or the language you translate to) google-scholar search for both terms. For Vertrauensbereich for example I get 7000, for Konfidenzintervall 37000 hits. Pretty clear which to select now.

JoKeyser replied:
@tomtomme, thank you for reconsidering, and helping with the translation process, there is still so much to do! Deciding between terms by counting search hits is a good idea, at least if its decisive :). I think in general, these kinds of questions just need to be discussed in the open, as we go along.
On Weblate, there is also a [glossary](https://hosted.weblate.org/glossaries/jasp/) that could help in the long run, and also to keep consistency across the modules. I now added "Konfidenzintervall".

An immediate follow-up question would be how to translate "credible" interval, which I started to translate as "Kredibilitätsintervall", which I never really liked. Wikipedia suggests to use [Glaubwürdigkeitsintervall](https://de.wikipedia.org/wiki/Glaubw%C3%BCrdigkeitsintervall), which seems better to me, in light of your argument toward beginner-friendliness? I also changed the glossary to reflect this.

tomtomme replied:
Yes. Glaubwürdigkeitsintervall is much much better. Kredibilitätsintervall has 100 hits on scholar, glaubwürdigkeitsintervall 25. But I think at this low volume JASP has a chance to shape the german language to a better way.

JoKeyser replied:
Okay, nice that we agree; now we will have to change this term in all translation strings. My overall impression is that many Bayesian terms don't have a settled translation. So it's especially important to keep discussing them, so JASP provides a good example :).

JoKeyser replied:
@tomtomme, just fyi, yesterday I replaced all instances of "Kredibilitätsinterall" with "Glaubwürdigkeitsintervall", and all instances of "Population" with "Grundgesamtheit", to match the (updated) glossary.
For some reason, Weblate sometimes takes hours/days to push these changes.

tomtomme committed:
Translated using Weblate (German)

Currently translated at 100.0% (4640 of 4640 strings)

Translation: JASP/JASP desktop
Translate-URL: https://hosted.weblate.org/projects/jasp/jasp-desktop/de/

EJWagenmakers commented:
I understand both perspectives. But if the field uses a particular term, then it would be odd for JASP to use something else, even though it might be more easy to understand initially. So I am totally OK with using the standard stats lingo -- this makes it easier for people to switch to JASP and recognize immediately  what output they can expect. 

In the help files, however, it makes perfect sense to mention more intuitive concepts. Basically, if in a particular country practitioners and educators use term X, then JASP also ought to use term X.

Cheers,
E.J.

tomtomme committed:
Translated using Weblate (German)

Currently translated at 100.0% (4640 of 4640 strings)

Translation: JASP/JASP desktop
Translate-URL: https://hosted.weblate.org/projects/jasp/jasp-desktop/de/

boutinb committed:
Translated using Weblate (French)

Currently translated at 99.9% (4854 of 4857 strings)

Translation: JASP/JASP desktop
Translate-URL: https://hosted.weblate.org/projects/jasp/jasp-desktop/fr/

boutinb committed:
Translated using Weblate (French)

Currently translated at 99.9% (4854 of 4857 strings)

Translation: JASP/JASP desktop
Translate-URL: https://hosted.weblate.org/projects/jasp/jasp-desktop/fr/

probalino committed:
Translated using Weblate (Spanish)

Currently translated at 6.8% (323 of 4745 strings)

Translation: JASP/JASP desktop
Translate-URL: https://hosted.weblate.org/projects/jasp/jasp-desktop/es/

Commit:
Translated using Weblate (Spanish)

Currently translated at 6.8% (323 of 4745 strings)

Translation: JASP/JASP desktop
Translate-URL: https://hosted.weblate.org/projects/jasp/jasp-desktop/es/

JoKeyser committed:
Translated using Weblate (German)

Currently translated at 100.0% (4640 of 4640 strings)

Translation: JASP/JASP desktop
Translate-URL: https://hosted.weblate.org/projects/jasp/jasp-desktop/de/

JoKeyser committed:
Translated using Weblate (German)

Currently translated at 86.0% (1855 of 2155 strings)

Translation: JASP/JASP engine
Translate-URL: https://hosted.weblate.org/projects/jasp/jasp-engine/de/
